### PR TITLE
migrateion: Add a case about migrate_tls_force

### PR DESCRIPTION
**Pull request by cliping**
XXXX-293816 - VM live migration - network data transport - TLS -
 migrate_tls_force

Signed-off-by: cliping <lcheng@redhat.com>

### CODE REVIEW

**Comment by cliping** [comment]
Test result:

# avocado run --vt-type libvirt --vt-machine-type q35 migration.network_data_transports.tls.migrate_tls_force
No python imaging library installed. Screendump and Windows guest BSOD detection are disabled. In order to enable it, please install python-imaging or the equivalent for your distro.
No python imaging library installed. PPM image conversion to JPEG disabled. In order to enable it, please install python-imaging or the equivalent for your distro.
JOB ID     : fd17f0fdd7c593d340707316af279e6fef632ebf
JOB LOG    : /var/lib/avocado/job-results/job-2022-06-14T12.23-fd17f0f/job.log
 (01/10) type_specific.io-github-autotest-libvirt.migration.network_data_transports.tls.migrate_tls_force.default_src_and_dest.p2p: STARTED
 (01/10) type_specific.io-github-autotest-libvirt.migration.network_data_transports.tls.migrate_tls_force.default_src_and_dest.p2p: PASS (242.57 s)
 (02/10) type_specific.io-github-autotest-libvirt.migration.network_data_transports.tls.migrate_tls_force.default_src_and_dest.non_p2p: STARTED
 (02/10) type_specific.io-github-autotest-libvirt.migration.network_data_transports.tls.migrate_tls_force.default_src_and_dest.non_p2p: PASS (242.37 s)
 (03/10) type_specific.io-github-autotest-libvirt.migration.network_data_transports.tls.migrate_tls_force.enable_src_disable_dest.p2p: STARTED
 (03/10) type_specific.io-github-autotest-libvirt.migration.network_data_transports.tls.migrate_tls_force.enable_src_disable_dest.p2p: PASS (249.67 s)
 (04/10) type_specific.io-github-autotest-libvirt.migration.network_data_transports.tls.migrate_tls_force.enable_src_disable_dest.non_p2p: STARTED
 (04/10) type_specific.io-github-autotest-libvirt.migration.network_data_transports.tls.migrate_tls_force.enable_src_disable_dest.non_p2p: PASS (250.44 s)
 (05/10) type_specific.io-github-autotest-libvirt.migration.network_data_transports.tls.migrate_tls_force.disable_src_enable_dest.p2p: STARTED
 (05/10) type_specific.io-github-autotest-libvirt.migration.network_data_transports.tls.migrate_tls_force.disable_src_enable_dest.p2p: PASS (249.88 s)
 (06/10) type_specific.io-github-autotest-libvirt.migration.network_data_transports.tls.migrate_tls_force.disable_src_enable_dest.non_p2p: STARTED
 (06/10) type_specific.io-github-autotest-libvirt.migration.network_data_transports.tls.migrate_tls_force.disable_src_enable_dest.non_p2p: PASS (249.97 s)
 (07/10) type_specific.io-github-autotest-libvirt.migration.network_data_transports.tls.migrate_tls_force.enable_src_and_dest.p2p: STARTED
 (07/10) type_specific.io-github-autotest-libvirt.migration.network_data_transports.tls.migrate_tls_force.enable_src_and_dest.p2p: PASS (249.49 s)
 (08/10) type_specific.io-github-autotest-libvirt.migration.network_data_transports.tls.migrate_tls_force.enable_src_and_dest.non_p2p: STARTED
 (08/10) type_specific.io-github-autotest-libvirt.migration.network_data_transports.tls.migrate_tls_force.enable_src_and_dest.non_p2p: PASS (249.59 s)
 (09/10) type_specific.io-github-autotest-libvirt.migration.network_data_transports.tls.migrate_tls_force.disable_src_and_dest.p2p: STARTED
 (09/10) type_specific.io-github-autotest-libvirt.migration.network_data_transports.tls.migrate_tls_force.disable_src_and_dest.p2p: PASS (248.39 s)
 (10/10) type_specific.io-github-autotest-libvirt.migration.network_data_transports.tls.migrate_tls_force.disable_src_and_dest.non_p2p: STARTED
 (10/10) type_specific.io-github-autotest-libvirt.migration.network_data_transports.tls.migrate_tls_force.disable_src_and_dest.non_p2p: PASS (246.93 s)
RESULTS    : PASS 10 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /var/lib/avocado/job-results/job-2022-06-14T12.23-fd17f0f/results.html
JOB TIME   : 2484.31 s

**Comment by cliping** [comment]
Depend on https://github.com/avocado-framework/avocado-vt/pull/3450.

**Review by fangge1212 — APPROVED** [review]

File: libvirt/tests/src/migration/migration_uri/migration_network_data_transport_tls.py [hunk 1]
```diff
@@ -11,19 +52,56 @@ def run(test, params, env):
...
+
+        """
+        test.log.info("Setup for migrate_tls_force.")
+
+        update_qemu_conf(params, test, remote_obj, local_obj)
+        migration_obj.setup_connection()
+
+    def cleanup_migrate_tls_force():
+        """
+        Cleanup steps for migrate_tls_foece
```
Comment (Yingshun) [inline libvirt/tests/src/migration/migration_uri/migration_network_data_transport_tls.py:67]: typo - ‘foece‘

**Review by Yingshun — COMMENTED** [review]
Others LGTM

File: libvirt/tests/src/migration/migration_uri/migration_network_data_transport_tls.py (hunk 1 above)
Comment (cliping) [reply libvirt/tests/src/migration/migration_uri/migration_network_data_transport_tls.py]: Updated, thanks.

**Review by Yingshun — APPROVED** [review]

**Pull request merged**